File: 9325.py
test = int(input())
total = 0
while test != 0 :
    price = int(input())
    total += price
    option = int(input())
    while option != 0 :
        opt = input().partition(" ")
        total += (int(opt[0]) * int(opt[2]))
        option -= 1
    test -= 1
    print(total)
    total = 0

File: 9325.c
#include <stdio.h>

int main(){
  int price, test, a, b, option;
  int total = 0;
  scanf("%d", &test);
  while ( test != 0){
    scanf("%d", &price);
    total += price;
    scanf("%d", &option);
    while (option != 0){
      scanf("%d %d", &a, &b );
      total += (a * b);
      option -= 1;
    }
    test -= 1;
    printf("%d\n", total);
    total = 0 ;
  }
}
